Source/SwiftLintFrameworkTests/TestHelpers.swift

```diff
@@ -29,15 +28,14 @@ extension XCTestCase {
             ruleDescription.triggeringExamples.flatMap({
                 violations($0, ruleDescription).map({$0.ruleDescription})
             }),
-            Array(count: ruleDescription.triggeringExamples.count, repeatedValue: ruleDescription))
+            Array(count: ruleDescription.triggeringExamples.count, repeatedValue: ruleDescription)
+        )
 
         if commentDoesntViolate {
```

Comment:
maybe should also test that, if a comment does violate, the violation still happens inside a comment?